### PR TITLE
[13.0][MIG] website_lazy_load_image and background image support

### PR DESCRIPTION
**Pull request by tarteo**
Also applies the fix proposed here: https://github.com/OCA/website/pull/719

### CODE REVIEW

**Review by simahawk — APPROVED** [review]
just code review, LG

**Review by ferranmateu — APPROVED** [review]

**Review by chienandalu — COMMENTED** [review]
Is this abandoned?

File: website_lazy_load_image/static/src/js/frontend.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+         * @private
+         */
+        _onFinishedAll: function() {
+            this.allFinished.resolve();
+            this.trigger("onFinishedAll");
+        },
+    });
+
+    require("web.dom_ready");
+    var lazy_image_loader = new LazyImageLoader(
```
Comment (Tardo) [inline website_lazy_load_image/static/src/js/frontend.js:100]: Can't use "web.public.widget"?

**Review by Tardo — CHANGES_REQUESTED** [review]

File: website_lazy_load_image/static/src/js/frontend.js (hunk 1 above)
Comment (tarteo) [reply website_lazy_load_image/static/src/js/frontend.js:100]: No

**Comment by simahawk** [comment]
@Tardo fine for you?
@tarteo travis is :red_circle:

**Pull request closed without merging**